### PR TITLE
Awesome book ES 6

### PR DESCRIPTION
**Pull request by ijosue16**
Created awesome books using ES 6 technology

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -202,12 +202,6 @@ You can deploy this project using:
 - GitHub: [@ijosue16](https://github.com/ijosue16)
```
Comment (julie-ify) [inline README.md:178]: - [x] You have duplicate **Built With** sections on your readme file. Kindly fix that.

- [x] Kindly include the key features of the application

  fhfd

File: index.js [hunk 2]
```diff
@@ -8,75 +12,24 @@ const addNew = document.querySelector('.Add-New');
...
-  static ContactRemove() {
-    formSubmision.style.display = 'none';
-    dataContainer.style.display = 'none';
-    contactSection.style.display = 'flex';
-    dataContainerTitle.style.display = 'none';
-  }
+function clock() {
+  const dt = DateTime.now();
+  time.innerHTML = dt.toLocaleString(DateTime.DATETIME_MED_WITH_SECONDS);
 }
```
Comment (julie-ify) [inline index.js]: - [x] Kindly refactor this to an arrow function in line with the ES6 syntax requirement.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @ijosue16,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by levy002 — COMMENTED** [review]
## Hello @ijosue16 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't run linters :red_circle: :red_circle:  :red_circle: 
image

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.js (hunk 2 above)
Comment (rloterh) [reply index.js]: well resolved! ✔️

File: README.md (hunk 1 above)
Comment (rloterh) [reply README.md:178]: well resolved! ✔️

Comment (ijosue16) [reply README.md:178]: i resolved it sir

File: README.md [hunk 3]
```diff
@@ -202,12 +176,6 @@ You can deploy this project using:
...
 
 
-**👤 Author2**
-
-- GitHub: [@maccony202](https://github.com/maccony202)
-
-
-
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (rloterh) [inline README.md:181]: - [ ] OPTIONAL
Kindly ensure that the _Future Features_ section on _line 184_ of your ReadMe includes 1-3 potential features to be added later.

File: README.md (hunk 1 above)
Comment (ijosue16) [reply README.md:178]: @rloterh  please check the README.md

File: README.md (hunk 3 above)
Comment (rloterh) [inline README.md:181]: - [ ] OPTIONAL
MIT license link _on line 252_ in your ReadMe is broken, kindly endeavor to fix it, and remove the guide note below it.

Comment (rloterh) [inline README.md:181]: - [ ] OPTIONAL
Kindly endeavor to fill the FAQ section with at least 2 Q&A(s), on _line 234_ of your ReadMe.

File: README.md [hunk 4]
```diff
@@ -80,41 +80,15 @@
...
 
 ### Key Features <a name="key-features"></a>
 
 > Describe between 1-3 key features of the application.
 
-- **[key_feature_1]**
-- **[key_feature_2]**
+- **[used modules]**
+- **[arrow functions]**
 - **[key_feature_3]**
```
Comment (rloterh) [inline README.md:93]: - [ ] OPTIONAL
Generally, the subheading text Author1 should be replaced with your full name in your ReadMe.

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Iradukunda,  

Your project is complete! There is nothing else to say other than... it's time to merge it 🍾 Congratulations!  🎉💯🌟 

## Highlights 👍  
- Good job making required changes

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

_Kindly consider my inline comments marked **[OPTIONAL]** under this review_

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (rloterh) [reply README.md:178]: @ijosue16 
Your project is complete! 
Cheers! 😃

**Pull request merged**